### PR TITLE
fix newlines on table

### PR DESCRIPTION
**Pull request by rachmadaniHaryono**
this pr is consist only fix for newline on table

---

- most diff from the actual pr are on readme file ([before](https://github.com/hardikvasa/google-images-download/blob/618d1c2cf8842b7d8adaa518109f542bc90e81a8/README.md), [after](https://github.com/rachmadaniHaryono/google-images-download/blob/d60692c2338a9d3c6e6b1d76199fa3d22655e5d0/README.rst)) and setup.py ([before](https://github.com/hardikvasa/google-images-download/blob/618d1c2cf8842b7d8adaa518109f542bc90e81a8/setup.py), [after](https://github.com/rachmadaniHaryono/google-images-download/blob/d60692c2338a9d3c6e6b1d76199fa3d22655e5d0/setup.py))
  - maybe squash the commits between https://github.com/hardikvasa/google-images-download/tree/618d1c2cf8842b7d8adaa518109f542bc90e81a8 and this pr after this pr is merged? 
- pypitest result can be found here https://test.pypi.org/project/google_images_download/

---
changes

- setup.py
  - removing rst generation. readme file already use rst
  - add mit license to classifier
- readme
  - change readme to rst
  - removing python2/3 usage section and example: with pip installation it is already a single command
    - it also mean not using `python` anymore on example.
  - removing bold effect on value in argument row
  - the explanation where the image downloaded

---

what is left

- change author_email in setup.py
- choose one license file. either `license.txt` or `license.md`
- either give remove the example to download similar image or add that feature to program

---

if you want i can also upload to pypi. just reply when everything is ready.

or you can also upload it yourself to pypi

### CODE REVIEW

**Comment by hardikvasa** [comment]
Thanks @rachmadaniHaryono 

Couple questions:
* `removing python2/3 usage section and example: with pip installation it is already a single command` I have never got this to work on my machine. here is what i did:

$ pip install google_images_download

when i go to my site-packages, i see the directory (google_images_download) in there. Then when i use $ googleimagesdownload [Arguments...]

i get the following error: -bash: googleimagesdownload: command not found

Am I not using it correctly? Should i be in a specific directory to use this command? is some path missing?

* Does pypi allow both licence.md and licence.txt? If yes, then i will just remove one of them

**Comment by rachmadaniHaryono** [comment]
>removing python2/3 usage section and example: with pip installation it is already a single command I have never got this to work on my machine. here is what i did:

>$ pip install google_images_download

>when i go to my site-packages, i see the directory (google_images_download) in there. Then when i use $ googleimagesdownload [Arguments...]

my guess you are on linux.

here is the same question, which may have the similar problem

https://askubuntu.com/questions/625354/installed-script-with-pip-run-like-a-command

tldr: check your $PATH

another alternative is using this command: `pip install -e .` on the project folder

to quote https://stackoverflow.com/a/30306403/1766261

>With `pip install -e for local` projects, the "SomeProject.egg-info" directory is created relative to the project path. This is one advantage over just using setup.py develop, which creates the "egg-info" directly relative the current working directory. More: [docs](https://pip.pypa.io/en/latest/reference/pip_install.html#editable-installs)

if you are developing this,  i recommend to use virtualenv or pyenv, or something similar (i use virtualenv due to habit). reason quoted from this link https://www.davidfischer.name/2010/04/why-you-should-be-using-pip-and-virtualenv/

>Virtualenv creates a Python environment that is segregated from your system wide Python installation. In this way, you can test your module without any external packages mucking up the result, add different versions of dependency packages and generally verify the exact set of requirements for your package.

>Does pypi allow both licence.md and licence.txt? If yes, then i will just remove one of them

i can't find which one is recommended. this guide that i use to upload to pypi using txt file

http://peterdowns.com/posts/first-time-with-pypi.html

also anything you use, change the setup.py like this https://stackoverflow.com/a/9977956/1766261

**Comment by rachmadaniHaryono** [comment]
>pip install google_images_download

that command still download the uploaded version from @psuzn, the latest version is not uploaded yet

**Comment by hardikvasa** [comment]
@rachmadaniHaryono 
I am actually testing it on macOS Sierra right now. i used the command `which python` and `which python3` and it showed me the path `/usr/local/bin/python`.

then i checked my path by (echo $PATH) and it shows `/usr/local/bin` as the part of the path. So i guess this is not the issue of the path...but maybe I am wrong.

I am just stressing on this because other users might face the same issue. That is why i had added to complete command in the README.

**Comment by hardikvasa** [comment]
Anyways, i will dig into the above issue a little later...merging this PR. Will make the required changes and update it here.

**Comment by rachmadaniHaryono** [comment]
https://github.com/tdryer/hangups/issues/222

Can you check it like that issue.

**Comment by hardikvasa** [comment]
@rachmadaniHaryono that actually worked, thanks! I will have to document this somewhere in the repo so other users do not have to go through the hassle.

one more question: should __init__.py just have 
`import google_images_download` instead of `import google_images_download.google_images_download` because it is in the same directory?

When i am now running the command after PIP install, i get the following error...complete stacktrace below:

```
googleimagesdownload -k 'teddy' -l 10

Item no.: 1 --> Item name = teddy
Evaluating...
Total Image Links = 100
Starting Download...
Completed ====> 1. 41eqT-kNHaL._SL500_AC_SS350_.jpg
Completed ====> 2. ae6a28a2-e959-4836-901e-0422d52878e0_1.2a2f5614103fad624505828fd755374b.jpeg.jpg
Completed ====> 3. 151922_7.jpg
Completed ====> 4. Biggest_6ft_Life_Size_huggable_Mocha_Brown_Teddy_Bear_Sunny_Cuddles__64682.1459535455.jpg
Completed ====> 5. Meddy-Teddy-Tree-Pose_1024x1024.png
URLError on an image...trying next one... Error: <urlopen error EOF occurred in violation of protocol (_ssl.c:661)>
Completed ====> 6. 91azUGbKJyL._SL1500_.jpg
Completed ====> 7. Cozy-Love-Cuddles-cream-teddy-bear-with-heart-38in__51768.1295498306.jpg
Completed ====> 8. Tabby-Toys-Brown-Teddy-Bear-SDL718079748-1-39ccd.jpg
Completed ====> 9. teddy-image-4.jpg
Completed ====> 10. teddy_and_silk_chocolate.jpg

Everything downloaded!
Total Errors: 1

Total time taken: 14.4074530602 Seconds
Traceback (most recent call last):
  File "/Library/Frameworks/Python.framework/Versions/2.7/bin/googleimagesdownload", line 11, in <module>
    sys.exit(main())
  File "/Library/Frameworks/Python.framework/Versions/2.7/lib/python2.7/site-packages/google_images_download/__init__.py", line 2, in main
    import google_images_download.google_images_download
ImportError: No module named google_images_download
```

which seems like because of the path in __init__.py

**Comment by hardikvasa** [comment]
> either give remove the example to download similar image or add that feature to program

can you elaborate on this? Are you referring to the --similar_images switch?
if yes, then that feature is already present in the code.

**Comment by rachmadaniHaryono** [comment]
your problem fixed should be fixed on new pr.

>can you elaborate on this? Are you referring to the --similar_images switch?
if yes, then that feature is already present in the code.

yes, but the doc on that is different, so i haven't changed it

```bash 
python3 pr.py -si <image url> -l 10
```

**Comment by hardikvasa** [comment]
@rachmadaniHaryono i have completed the changes discussed above. Can you update the repo on pypi (1.0.2)?

Also can you or @psuzn give me access to the pypi repo so i can update it going forward?

Thanks!

**Comment by psuzn** [comment]
@hardikvasa  can you give me your pypi username?

**Comment by rachmadaniHaryono** [comment]
@psuzn, after I check it, he already got it.

@hardikvasa, please edit the recent commit. There is still some mistake there

**Comment by hardikvasa** [comment]
@psuzn thanks, I got the access.
@rachmadaniHaryono  in the recent commit? Can you enlist the things that need editing?

**Comment by rachmadaniHaryono** [comment]
@hardikvasa https://github.com/hardikvasa/google-images-download/commit/6efc092efb35be7fad00e65a4c0f49ff84d5179e

**Comment by hardikvasa** [comment]
@rachmadaniHaryono are you referring to the `sudo` thing put into the troubleshooting guide?
I was thinking of putting a 'warning link' below that rather than removing it since i had a similar issue while installing the repo and sudo solved it. Seems like virtualenv should be an alternative to using the elevated permissions...

**Comment by rachmadaniHaryono** [comment]
you can use `--user` flag. it is the easiest and don't require virtualenv

**Comment by hardikvasa** [comment]
@rachmadaniHaryono sounds good. I will change the doc this weekend along with other feature updates.
